### PR TITLE
Linux Tutorial added and cleaned up the pull requests! -Thank you for your help!

### PR DESCRIPTION
**Pull request by Ry10p**

### CODE REVIEW

**Comment by Ry10p** [comment]
If I did anything wrong please let me know I want to do the best job I can do!

**Comment by Ry10p** [comment]
Also....

I am going to school for GIS, and am VERY interested in the toga-django project.

My Teacher Dr. Nara would like me to pick a project to work on for the semester.  

Is there anything that needs (or can) be done with that project?

I am interested in adding database functionality!

Thank you,
Ryan Carl

File: docs/tutorials/tutorial-0-Linux.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+Linux Ubuntu 16.04 Tutorial 0 - Hello, world!
+===============================================
```
Comment (freakboy3742) [inline docs/tutorials/tutorial-0-Linux.rst]: You've still got some overflow on the underlining; and the title is still Ubuntu-specific.

File: docs/tutorials/tutorial-0-Linux.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+In this tutorial, you'll take a really simple "Hello, world!" program written in
+Python, convert it into a working Ubuntu Linux project.
```
Comment (freakboy3742) [inline docs/tutorials/tutorial-0-Linux.rst]: Again - there's nothing Ubuntu specific here. The text should refer to this being a "Linux" tutorial, with a note somewhere near the top (around this spot would be fine) that reads something like this:

```
.. note:
    This tutorial has been tested with Ubuntu 16.04; however it should work on any Linux.
```

File: docs/tutorials/tutorial-0-Linux.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+Setup
+-----
+
+This tutorial assumes you've read and followed the instructions in
+:doc:`/intro/getting-started`. This Tutorial is for Linux Users developing for Ubuntu 16.04!
```
Comment (freakboy3742) [inline docs/tutorials/tutorial-0-Linux.rst]: There's a flow problem here - this sentence doesn't lead into the bullet points below.

**Comment by freakboy3742** [comment]
@Ry10p As for working on toga for a semester project - There's plenty that needs to be done, and every little bit of help is gratefully accepted. 

If you're looking for an area to contribute, I'd suggest toga-gtk is possibly a better place to start - toga-django is still very much in an "early-prototype" state, and there's a lot to wrap your head around. toga-gtk will be a lot more approachable, and the todo list is a lot better defined.

If you're looking for something that is specifically in the web space, contributing to Batavia may be a better place to start than toga-django. Batavia is a key part of making toga-django work, and it still has a lot of work. The Batavia documentation has a contribution guide, and there's a [guide to your first Batavia contribution](https://pybee.org/contributing/how/first-time/what/batavia/) on our website.

**Comment by Ry10p** [comment]
@freakboy3742 ,

AMAZING!! Python really got me interested in school and I LOVE it!

I think I have made the needed changes required to the tutorial-0-Linux page.

Ill be getting to work on over the summer, so ill have plenty of time to dive into the docs and get to work on the AMAZING toga project!

Thanks again,
Ryan Carl

**Comment by Ry10p** [comment]
Is there an issue on toga-gtk that could be good to sink my teeth in (for a first time contributor)?

Many of the issues have work already pending on them and I don't want to upset anyone...

I have taken a few python classes in school and feel comfortable to try helping out!

**Comment by freakboy3742** [comment]
The biggest task is to port widgets. The Cocoa backend is the most complete Toga backend; so there are widgets defined on the Cocoa backend that haven't been implemented on Gtk. So - implement them :-)

As for standing on people's toes - if you see a specific ticket that looks interesting, drop a comment on it saying that you'd like to work on it, and ask for a progress update. If you don't get a response, or they say "Go right ahead", then... go right ahead :-)

**Comment by Ry10p** [comment]
Thank you so much!

**Pull request merged**